src/app/signup/signup.component.ts

```diff
@@ -0,0 +1,57 @@
+import { AppserviceService } from './../appservice.service';
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-signup',
+  templateUrl: './signup.component.html',
+  styleUrls: ['./signup.component.css']
+})
+export class SignupComponent implements OnInit {
+
+  firstname:any
+  email:any
+  password:any
+  cpassword:any
+  successMsg:any
+
+  constructor(private http:HttpClient,private service:AppserviceService) { }
+
+  ngOnInit(): void {
+  }
+
+  signup(){
+    if(this.cpassword===this.password){
+      let body={
+
+        "firstName": "Fourth",
```

Comment:
why static body?

Reply:
from front end  we are taking only 3-4  values (as instructed by Harsha to avoid large signup form) but backend is expecting more than that , so passing these static values will help in login